checks/apps/gromacs/gromacs_benchpep-h.py

```diff
@@ -0,0 +1,174 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+from hpctestlib.sciapps.gromacs.benchmarks import gromacs_check
+
```

Comment:
Ideally we should add some documentation here on what is the aim, what run configs are being tested.